core/domain/draft_upgrade_services.py

```diff
@@ -312,6 +312,25 @@ def _convert_html_in_draft_change_list(
                 })
         return draft_change_list
 
+    @classmethod
+    def _convert_states_v52_dict_to_v53_dict(
+        cls, draft_change_list: List[exp_domain.ExplorationChange]
+    ) -> List[exp_domain.ExplorationChange]:
+        """Converts draft change list from state version 52 to 53. State
+        version 52 adds catchMisspellings customization_arg to TextInput
+        interaction. As this is a new property and therefore
+        doesn't affect any pre-existing drafts, there should be
+        no changes to drafts.
```

Comment:
Hmm, what if some draft contains TextInput interaction?

Reply:
Then the interaction will not contain anything, right?

Reply:
I actually did it in accordance with what is done in other migrations that add customization_args. For example, [here](https://github.com/oppia/oppia/blob/95f5f83b5a17206072cb76590bc6df87c70f2f9a/core/domain/draft_upgrade_services.py#L417)

Reply:
But these might have been done wrong.

Reply:
Can you please investigate?

Reply:
After investigating, I think adding a new property should not affect the drafts. The interaction would just be migrated with the new property with default values.

Reply:
> The interaction would just be migrated with the new property with default values.

Interaction in a draft? Why?

Reply:
Yaa, I just checked locally, the TextInput interaction in a draft is migrated.

Reply:
Hmm, okay interesting. Can you please explain how that happens?

Reply:
I'm not really sure how that happens. However, I did create an exploration draft on develop after opening the local server with the `--save_datastore` flag. After changing to a different branch, the draft had been migrated with the new field.

Reply:
Okay, thanks

Reply:
I think this needs to be investigated, we should make sure that the customization args are populated with the correct value.

Also, AFAIK we don't change the change_list in the backend (except in the draft_updagrade_srevice layer) so I was not expecting the changes list to be auto-populated with new customization_arg!

Reply:
So sorry to miss the comment. I actually did follow what happened in other migrations involving customization_arg and checked it by performing the migration like mentioned in the last comment in which the args are being populated with the default value. I actually don't really understand the exact issue too.

Reply:
@vojtechjelinek Can you please help get this comment resolved?